src/sentry/lang/javascript/processor.py

```diff
@@ -210,16 +210,23 @@ def fetch_release_file(filename, release):
             filename_idents.append(ReleaseFile.get_ident(filename_path))
 
         try:
-            releasefile = ReleaseFile.objects.filter(
+            possible_files = ReleaseFile.objects.filter(
                 release=release,
                 ident__in=filename_idents,
-            ).select_related('file').get()
+            ).select_related('file')
         except ReleaseFile.DoesNotExist:
```

Comment:
This `except` will never get hit now, so you'll  have to do it youself by checking `len(possible_files) == 0`.